packages/dear-imgui/cpp/reactimgui.cpp

```diff
@@ -111,11 +119,41 @@ void ReactImgui::InitWidget(const json& widgetDef) {
 
         m_widgets[id] = m_widget_init_fn[type](widgetDef, this);
         m_hierarchy[id] = std::vector<int>();
+
+        if (type == "Table") {
+            const std::lock_guard<std::mutex> lock(m_tableSubjectsMutex);
+
+            m_tableSubjects[id] = rpp::subjects::replay_subject<TableData>{100};
+
+            auto handler = std::bind(&ReactImgui::HandleTableData, this, id, std::placeholders::_1);
+
+            m_tableSubjects[id].get_observable() | rpp::ops::buffer(50) | rpp::ops::subscribe(handler);
```

Comment:
I'm having a hard time 'translating' RxJS into RPP equivalent code. In essence, I'd actually like to buffer observables and emit a consolidated chunk every 100ms.
One thing I have noticed with this approach is that only the initial batch consists of 50 items, subsequent calls to ReactImgui::HandleTableData() are made with vectors containing exactly 1 item.